browser/main/modals/PreferencesModal/index.js

```diff
@@ -133,7 +142,8 @@ class Preferences extends React.Component {
       {target: 'INFO', label: i18n.__('About')},
       {target: 'CROWDFUNDING', label: i18n.__('Crowdfunding')},
       {target: 'BLOG', label: i18n.__('Blog'), Blog: this.state.BlogAlert},
-      {target: 'SNIPPET', label: i18n.__('Snippets')}
+      {target: 'SNIPPET', label: i18n.__('Snippets')},
+      {target: 'IMPEXP', label: i18n.__('Import/Export'), ImpExp: this.state.ImpExpAlert}
```

Comment:
I think `Import/Export` is not a very suitable name, because in this case we only export config and nothing else. Do you have any idea for another name?

Reply:
Maybe 'Backup Config', 'Backup Settings', or 'Backup Preferences'